parser/sql/dialect/hive/src/main/antlr4/imports/hive/DDLStatement.g4

```diff
@@ -35,6 +35,11 @@ use
     : USE (identifier | DEFAULT)
     ;
 
+createTable
+    : CREATE createTableSpecification? TABLE ifNotExists? tableNameWithDb createDefinitionClause? tableComment? partitionedBy? clusteredBy? skewedBy? rowFormat? storedAs? storageLocation? tblProperties? (AS select)?
+    | CREATE createTableSpecification? TABLE ifNotExists? tableNameWithDb LIKE existingTableName storageLocation?
```

Comment:
Can you merge the same `CREATE createTableSpecification? TABLE ifNotExists? tableNameWithDb`